Dolphin scripts/Entrance Randomizer/__main__.py

```diff
@@ -38,15 +37,14 @@
     state,
 )
 
-remove_disabled_exits()
 set_transitions_map()
 randomize_shaman_shop()
 
-# This is necessary until/unless I map all areas even those not randomized.
+# This is necessary as long as the player can choose an unused level (which isn't in the json)
 try:
     starting_area_name = TRANSITION_INFOS_DICT[starting_area].name
 except KeyError:
-    starting_area_name = hex(starting_area).upper() + " (not in randomization)"
+    starting_area_name = hex(starting_area).upper() + " (not in transition_infos.json)"
```

Comment:
This is meant to be very-much user facing, not sure that `transition_infos.json` will mean much to the end-user, and it might sound like they're missing something.

Since you've added all levels to the transition infos, I don't think this is supposed to happen anymore?
But keeping this condition around for data validation doesn't seem like a bad idea.
So it's true that `not in randomization` isn't a super relevant message anymore.

I was doing some tests yesterday and thought that whenever we can't find the level name (for instance "current level" or "previous level" in blue text on screen), we might ask the user to report it. Especially relevant for levels that use weird entrance IDs (like jungle canyon and BBCamp).

Maybe a message like `" (not a known level)"` or `" (unknown)"` might fit better here? What do you think?

Reply:
Currently the only possible way for this to give a KeyError is if the user manually chose a starting area that is not in the `transition_infos.json`. And given that I added all "normal" levels to the json that only leaves the other unused levels.
And I think it's kinda silly to have the program crash just because the user wanted to try out an unused level.
So I think we should still keep this line.
As for what text we put in it, I'm fine with `" (unknown level)"` for instance.
If you're fine with that I'll change that in the next commit.

Reply:
`" (unknown level)"` sounds good to me.